definitions/divorce/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -799,12 +799,23 @@
     "PageDisplayOrder": 1,
     "ShowSummaryChangeOption": "Yes"
   },
+  {
+    "LiveFrom": "23/02/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "updateContactDetails",
+    "CaseFieldID": "DerivedPetitionerSolicitorAddr",
+    "PageFieldDisplayOrder": 44,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "updateContactDetails",
```

Comment:
PageID and eventId can be the same?

Reply:
I guess so, you can see it's the same for the other events below